src/scripts/main.js

```diff
@@ -1 +1,84 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function message(classMessage, textMessage) {
+  const newMessage = document.createElement('div');
+
+  newMessage.classList.add(classMessage);
+  newMessage.dataset.qa = 'notification';
+  newMessage.innerText = textMessage;
+  body.append(newMessage);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  body.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+
+    resolve(
+      {
+        classMessage: 'success',
+        textMessage: 'First promise was resolved',
+      }
+    );
```

Comment:
this objects it duplicated, so it's better to create separate function with instance of this objects, and set messages as params, for instance:
```
resolve(resultText('success', 'First promise was resolved'));
    
function getResultText(classMessage, textMessage) {
  return { classMessage, textMessage };
}
```